src/patroni.py

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env python3
+# Copyright 2021 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+"""Helper class used to manage interactions with Patroni API and configuration files."""
+
+import logging
+import os
+import pwd
+
+import requests
+from jinja2 import Template
+from tenacity import retry, stop_after_attempt, wait_exponential
+
+logger = logging.getLogger(__name__)
+
+
+STORAGE_PATH = "/var/lib/postgresql/data"
```

Comment:
Same as before. Would be nice to have a single place for this constant

Reply:
I agree. Thanks Paulo!